gradle.properties

```diff
@@ -3,14 +3,13 @@
 # their own projects. So don't get fancy with syntax!
 
 org.gradle.java.installations.auto-download=false
-org.gradle.jvmargs=-Dfile.encoding=UTF-8
 
 # Versioning
 cordaProductVersion = 5.1.0
 # NOTE: update this each time this module contains a breaking change
 ## NOTE: currently this is a top level revision, so all API versions will line up, but this could be moved to
 ##   a per module property in which case module versions can change independently.
-cordaApiRevision = 747-MGM
+cordaApiRevision = 5-MGM
```

Comment:
that looks a bit strange to me - how come we're decreasing the version?

Is it the case it was reset down to 1 (due to the bump of the major version to 5.1) at some point and we hadn't synced that yet? Or something else?

Reply:
Yes, it was reset : https://github.com/corda/corda-api/commit/6028bb677d518f3cfbcc4f8dda7a917fe0a85bda
I guess we haven't had that yet or kept using 74+-MGM when we did the last sync.